### PR TITLE
[AIRFLOW-5186] Move GCP Translate to core

### PR DESCRIPTION
**Pull request by turbaszek**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-5186

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:
This PR moves GCP Translate from contrib to core. For more information check AIP-21.

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by turbaszek** [comment]
Depends on #5791.

File: airflow/gcp/example_dags/example_speech.py [hunk 1]
```diff
@@ -29,9 +29,9 @@
 
 from airflow.utils import dates
 from airflow import models
-from airflow.contrib.operators.gcp_text_to_speech_operator import GcpTextToSpeechSynthesizeOperator
-from airflow.contrib.operators.gcp_speech_to_text_operator import GcpSpeechToTextRecognizeSpeechOperator
-from airflow.contrib.operators.gcp_translate_speech_operator import GcpTranslateSpeechOperator
+from airflow.gcp.operators.text_to_speech import GcpTextToSpeechSynthesizeOperator
+from airflow.gcp.operators.speech_to_text import GcpSpeechToTextRecognizeSpeechOperator
```
Comment (kaxil) [inline airflow/gcp/example_dags/example_speech.py:33]: Maybe we should probably have a single file for `speech_to_text` & `text_to_speech` ???

Comment (turbaszek) [reply airflow/gcp/example_dags/example_speech.py:33]: There are two product / APIs:
- Cloud Text-to-Speech API
- Cloud Speech-to-Text API

We could put those under "conversation" like here:
https://cloud.google.com/products/ai/?tab=tab3
but I am not sure if user will know where to look for the operators.

**Review by kaxil — APPROVED** [review]

File: airflow/gcp/example_dags/example_speech.py (hunk 1 above)
Comment (kaxil) [reply airflow/gcp/example_dags/example_speech.py:33]: 👍 Lets keep it as it is in that case

**Review by swalkowski — COMMENTED** [review]
Why don't we keep gcp_translate_speech_operator.py as deprecated, just like we do for other operators?

Otherwise, LGTM.

**Comment by turbaszek** [comment]
Very good catch! I am already working on tests for new imports and found this file missing.

**Pull request merged**